packages/nodes-base/nodes/Postgres/Postgres.node.functions.ts

```diff
@@ -1,6 +1,6 @@
 import { IDataObject, INodeExecutionData } from 'n8n-workflow';
 import pgPromise from 'pg-promise';
-import pg from 'pg-promise/typescript/pg-subset';
+import * as pg from 'pg-promise/typescript/pg-subset';
```

Comment:
With `esModuleInterop` enabled, namespace import here should in theory be unnecessary.

Reply:
Updated to import `{ IClient }` from pg-subset directly.